### ISSUE TITLE
Developer link not in footer

### ISSUE THREAD

**Issue by JRobertEdwards** [Say-Their-Name/say-their-names-web#154]
FILL OUT THE FORM BELOW OR THE ISSUE WILL BE AUTO-CLOSED

**Issue Type (check one)**

- [ ] Bug Report
- [x ] Feature Idea
- [ ] Documentation
- [ ] Technical Discussion

**Description (Please type a clear and concise description below)** 

When looking through the app it might be convenient to have a link for the repo's on the site footer for better user experience.

**Comment by Easybuoy** [comment]
We have that on the About page already @JRobertEdwards

**Comment by JRobertEdwards** [comment]
I'm aware it's in that page however it is not obvious to a visitor of the site that contribution is possible unless that page is visited. More visibility would reach more people @Easybuoy

**Comment by vinny2020** [comment]
I like this idea, a simple github icon with "Contribute" would do the trick

**Comment by Easybuoy** [comment]
Yeah, I thought I responded to your message @JRobertEdwards, I messaged Michael Sarpong (`ui-ux`) team to get his thoughts, but he hasn't responded. I'd send to the `ui-ux` channel to get their thoughts and maybe design how it would look like.

**Comment by alfalcon90** [comment]
Updated footer: https://www.figma.com/file/AUlfUejB2SVF8ZHSzxne7O/STNS?node-id=1135%3A1017
![Screenshot from 2020-07-13 17-33-44](https://user-images.githubusercontent.com/56562400/87367175-2306bb80-c52f-11ea-99cf-99603253b65f.png)

**Comment by Sharkesm** [comment]
@alfalcon90 Looks neat 🚀

**Comment by JRobertEdwards** [comment]
Looks great! @alfalcon90

**Comment by JRobertEdwards** [comment]
I've just been working on this this evening but having issues with getting into the Slack channel to ask. I've not been able to find the icons used in your updated footer although they are slightly different to the icons on the current site, should they all be converted to the non-filled versions?
